microsoft-edge/devtools-guide-chromium/whats-new/2023/07/devtools-115.md

```diff
@@ -0,0 +1,129 @@
+---
+title: What's new in DevTools (Microsoft Edge 115)
+description: "Improvements for connecting DevTools to Azure Artifacts symbol server. Snapshotting improvements for the Memory tool. Refreshing the DevTools icons and buttons. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/18/2023
+---
+# What's New in DevTools (Microsoft Edge 115)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Improvements for connecting DevTools to Azure Artifacts symbol server
+
+<!-- Subtitle: You can now use Azure Active Directory for authentication instead of Personal Access Tokens (PATs). You can connect to multiple symbol servers and filter which sourcemaps you want DevTools to fetch. -->
+
+In Microsoft Edge 115, it's easier and more customizable to connect DevTools to Azure Artifacts symbol server.  In previous versions of Microsoft Edge, to authenticate to the symbol server, you had to generate Personal Access Tokens (PATs) in Azure DevOps and then paste them into DevTools. Now, as long as you're logged into Microsoft Edge, you can connect to the symbol server by using Azure Active Directory.
```

Comment:
I believe this requires a business account. I don't believe `as long as you're logged in` is adequate. I don't know what the proper name is for this, however.

Reply:
@robpaveza - probably "work account", not "business account", based on full-text search of this repo.

Reply:
or maybe should be:
using a Microsoft account (MSA) that's a work account

Reply:
replaced this comment thread by fresh comment https://github.com/MicrosoftDocs/edge-developer/pull/2654/files#r1303525933